src/ctype/CTypeMetadata.spec.ts

```diff
@@ -0,0 +1,51 @@
+import CType from './CType'
+import ICType, { ICtypeMetadata } from '../types/CType'
+import CTypeMetadata from './CTypeMetadata'
+import * as CTypeUtils from './CTypeUtils'
+import { CTypeWrapperMetadata } from './CTypeSchema'
+
+jest.mock('../blockchainApiConnection/BlockchainApiConnection')
+
+describe('CType', () => {
+  const ctype = new CType({
+    schema: {
+      $id: 'http://example.com/ctype-1',
+      $schema: 'http://kilt-protocol.org/draft-01/ctype#',
+      properties: {
+        'first-property': { type: 'integer' },
+        'second-property': { type: 'string' },
+      },
+      type: 'object',
+    },
+  } as ICType)
+
+  const ctypeMetadata = {
+    metadata: {
+      $id: 'http://example.com/metadata-1',
+      $schema: 'http://kilt-protocol.org/draft-01/ctype#',
+      title: { type: 'string' },
+      description: { type: 'string' },
+      properties: {
+        'first-property': { type: 'integer' },
+        'second-property': { type: 'string' },
+      },
+      type: 'object',
+    },
+    ctypeHash: 'string',
+  } as ICtypeMetadata
+
+  const metadata = new CTypeMetadata(ctype, ctypeMetadata)
+
+  it('verifies the metadata of a ctype', async () => {
+    expect(metadata.ctypeHash).not.toHaveLength(0)
```

Comment:
It seems pointless, but necessary at the same time.

Reply:
It is fine